node/api/renter.go

```diff
@@ -178,62 +175,81 @@ func (api *API) renterHandlerGET(w http.ResponseWriter, req *http.Request, _ htt
 
 // renterHandlerPOST handles the API call to set the Renter's settings.
 func (api *API) renterHandlerPOST(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
-	// Scan the allowance amount.
-	funds, ok := scanAmount(req.FormValue("funds"))
-	if !ok {
-		WriteError(w, Error{"unable to parse funds"}, http.StatusBadRequest)
-		return
-	}
+	// Get the existing settings
+	settings := api.renter.Settings()
 
+	// Scan the allowance amount. (optional parameter)
+	if f := req.FormValue("funds"); f != "" {
+		funds, ok := scanAmount(req.FormValue("funds"))
```

Comment:
declaring a local variable (`f`) seems to hurt more than it helps. The lookup isn't costly, so the only reason to avoid multiple calls to `req.FormValue` is verbosity.

Reply:
Yep, that's the only reason why I did it. I don't like to have `"funds"` twice in 2 lines. If you want I can change that back. I missed replacing it with `f` in this line.